src/controllers/controller.bar.js

```diff
@@ -286,6 +286,14 @@ export default class BarController extends DatasetController {
 
 		for (i = 0; i < ilen; ++i) {
 			item = metasets[i];
+
+			if (
+				typeof dataIndex !== 'undefined' &&
+				isNullOrUndef(me.chart.data.datasets[item.index].data[dataIndex])
```

Comment:
Should we use the parsed value here?

Reply:
Good question. We probably should, I think if the parsing fails we'd get a null. Also, as written I realize my code wont work with object data where the `y` value is null